src/PowerShellEditorServices.Host/PowerShellEditorServices.Host.csproj

```diff
@@ -20,6 +20,8 @@
     <PackageReference Include="Microsoft.PowerShell.SDK">
       <Version>6.0.0-alpha13</Version>
     </PackageReference>
+    <PackageReference Include="System.Runtime.Extensions" Version="4.3.0" />
+    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
```

Comment:
Now that we are shipping this, I think there were a couple instances where we did some ifdefs between Core and Framework (which didn't have this package). If it's easy, can you remove the ifdefs

Reply:
Also don't forget to register the open source usage.